utils/db.py

```diff
@@ -92,12 +93,14 @@ class Db(Mapping):
         tables, like the mapping interface or :py:meth:`values`.
 
     """
-    def __init__(self, hostname=None, credentials=None):
+    def __init__(self, hostname=None, credentials=None, db_port=None):
         self._table_cache = {}
-        if hostname is None:
+        if hostname is None and db_port is None:
```

Comment:
@psav  I proposed to @quarckster so it could be possible to extend the Appliance object to also take the database IP and then have some db_address property which would resolve to either db address and if not specified, then to the appliance IP. In next iterations, it would be possible to access the DB IP different way for non-appliance databases ...